docs/diagrams/GroupState1.puml

```diff
@@ -0,0 +1,15 @@
+@startuml
+!include style.puml
+skinparam ClassFontColor #000000
+skinparam ClassBorderColor #000000
+skinparam ClassBackgroundColor #FFFFAA
+
+title Current state before modification
+
+package addressBook <<rectangle>> {
+    class State1 as "<u>p0:Person</u>\nstudentId:A0123456X"
+    class State2 as "<u>p1:Person</u>\nstudentId:A1234567H"
```

Comment:
Should the attributes be in a separate column from the class name and object name, and values indicated using `=` instead of `:`?

Reply:
I see what you mean. I'll make those changes.

Reply:
The changes are done.